pkg/resource/interfaces.go

```diff
@@ -84,6 +84,12 @@ type Reclaimer interface {
 	GetReclaimPolicy() v1alpha1.ReclaimPolicy
 }
 
+// A Deletable may specify a DeletionPolicy.
+type Deletable interface {
```

Comment:
`Deletable` kind of implies that if the resource doesn't satisfy this interface, it can't be deleted. I don't really have great suggestions though. Maybe `HasDeletionPolicy`?

Reply:
I'm open to other suggestions, but I prefer `Deletable` to `HasDeletionPolicy`. Would `Orphanable` be preferable?

Reply:
I went with `Orphanable`.